src/uct/sm/mm/base/mm_iface.c

```diff
@@ -163,7 +163,8 @@ static ucs_status_t uct_mm_iface_query(uct_iface_h tl_iface,
                                           UCT_IFACE_FLAG_AM_BCOPY            |
                                           UCT_IFACE_FLAG_PENDING             |
                                           UCT_IFACE_FLAG_CB_SYNC             |
-                                          UCT_IFACE_FLAG_CONNECT_TO_IFACE;
+                                          UCT_IFACE_FLAG_CONNECT_TO_IFACE    |
+                                          UCT_IFACE_FLAG_ERRHANDLE_PEER_FAILURE;
```

Comment:
pls, don't add support for PEER_FAILURE in XPMEM
it doesn't support PEER_FAILURE and it's hard to add it, because XPMEM touches remote memory which doesn't exist if a peer was killed prior.

Reply:
ok,
the assumption was that posix/sysv would be the ones who detect the failure.
Since now I'll add support for cma and knem I'll remove this cap for xpmem